Dockerfile

```diff
@@ -1,28 +1,33 @@
 FROM quay.io/azavea/spark:0.1.0
 
-RUN apt-get update && apt-get install -y git --no-install-recommends
+MAINTAINER Azavea <systems@azavea.com>
 
-ENV SPARK_JOBSERVER_VERSION master
+ENV SPARK_JOBSERVER_VERSION 0.5.2
 ENV SPARK_JOBSERVER_HOME /opt/spark-jobserver
+ENV SPARK_JOBSERVER_BUILD /tmp/spark-jobserver
+ENV LEIN_ROOT true
 
-RUN mkdir -p ${SPARK_JOBSERVER_HOME}
-RUN git clone https://github.com/spark-jobserver/spark-jobserver.git /tmp/spark-jobserver
+RUN mkdir -p ${SPARK_JOBSERVER_HOME} ${SPARK_JOBSERVER_BUILD} \
+  && wget -qO /usr/local/bin/lein https://raw.githubusercontent.com/technomancy/leiningen/stable/bin/lein \
+  && chmod a+x /usr/local/bin/lein \
+  && lein
 
-WORKDIR /tmp/spark-jobserver
+WORKDIR ${SPARK_JOBSERVER_BUILD}
 
-RUN sbt ++${SCALA_VERSION} job-server-extras/assembly
+COPY etc/project.clj ${SPARK_JOBSERVER_BUILD}/
 
-RUN cp job-server-extras/target/scala-${SCALA_MAJOR_VERSION}/spark-job-server.jar \
-       job-server/config/log4j-server.properties ${SPARK_JOBSERVER_HOME} \
-  && touch ${SPARK_JOBSERVER_HOME}/settings.sh
+RUN lein uberjar \
```

Comment:
Yeah, this change is questionable, but I'm not really aware of a cleaner way to build a JAR our of a few existing JARs. @lossyrob suggested that it could be done via `sbt` with the assembly plugin too, but I didn't get to far with that.